apps/backend/src/mutations/ProposalMutations.ts

```diff
@@ -380,13 +380,21 @@ export default class ProposalMutations {
   @Authorized([Roles.USER_OFFICER])
   async notify(
     user: UserWithRole | null,
-    { proposalPk }: { proposalPk: number }
+    {
+      proposalPk,
+      ignoreNotifiedFlag,
+    }: { proposalPk: number; ignoreNotifiedFlag: boolean }

```

Comment:
suggestion: maybe you can use the datatype we have already defined in `apps/backend/src/resolvers/mutations/NotifyProposalMutation.ts` 
`NotifyProposalArgs` , so we don't have to define it twice